aesara/tensor/random/basic.py

```diff
@@ -154,6 +179,52 @@ def __call__(self, scale=1.0, size=None, **kwargs):
 exponential = ExponentialRV()
 
 
+class WeibullRV(RandomVariable):
```

Comment:
This distributions extends from the numpy version by including the beta parameter. I added it mostly because then we can use it directly on the PyMC3 side, but if it feels strange in here, I will happily remove it. I made the default beta=1, that would be equivalent to the numpy version.

Reply:
We don't want to deviate from the NumPy interface.